src/redux/actions/like.js

```diff
@@ -0,0 +1,36 @@
+import { api } from '../../utils/api';
+
+import {
+  LIKE_SUCCESS, DISLIKE_SUCCESS, GET_LIKES
+} from '../../constants';
+
+export const likeSuccess = () => ({
+  type: LIKE_SUCCESS,
+});
+
+export const dislikeSuccess = () => ({
+  type: DISLIKE_SUCCESS,
+});
+
+export const getTheLikes = payload => ({
```

Comment:
would `getLikes` be a better name for this action creator?

Reply:
I have another function called getLikes so I was trying to avoid confusion.